packages/web/src/app/doc-pages/components/tag-doc/tag-data-colored-ceg/tag-data-colored-ceg.component.scss

```diff
@@ -0,0 +1,6 @@
+:host {
+  display: flex !important;
```

Comment:
I was just wondering, does this not work without `!important`?

Reply:
No, there's a `display: contents` from somewhere else that has a higher specificy. However I'll just change this to instead have a `<div class="e-flex e-gap-16 e-flex-wrap">` as a wrapper, so that it's clear to see what is happening in the code example